### PR TITLE
Fix DLNA/UPNP port, and expose them in Docker

### PR DESCRIPTION
**Pull request by randomnicode**
In response to and possible fix for https://github.com/airsonic/airsonic/issues/1379

- Fixes the listening port to 4041 for UPNP/DLNA (otherwise an ephemeral port is auto-assigned)
  - Can be customized by a JVM variable `UPNP_PORT`
    - E.g. `java -DUPNP_PORT=4042 -jar airsonic.war`
- Exposes tcp port 4041 in Dockerfile
- Exposes udp port 1900 in Dockerfile (needed for SSDP in UPnP)
- Removes an extraneous class

Note that if the default port is modified using `UPNP_PORT`, then user needs to expose and bind the custom port manually if running a container.

### CODE REVIEW

File: airsonic-main/src/main/java/org/airsonic/player/service/UPnPService.java [hunk 1]
```diff
@@ -61,10 +62,17 @@
...
+    
     private UpnpService upnpService;
+    
     @Autowired
     @Qualifier("dispatchingContentDirectory")
     private CustomContentDirectory dispatchingContentDirectory;
+    
+    @Autowired
+    @Value("#{ systemProperties['UPNP_PORT'] ?: T(org.airsonic.player.service.SettingsService).DEFAULT_UPNP_PORT }")
+    private int port;
```
Comment (muff1nman) [inline airsonic-main/src/main/java/org/airsonic/player/service/UPnPService.java]: This doesn't match the existing methodology for managing settings. Should instead use a getter through settingservice.

Comment (randomnicode) [reply airsonic-main/src/main/java/org/airsonic/player/service/UPnPService.java]: `T(org.airsonic.player.service.SettingsService).DEFAULT_UPNP_PORT`

is the public variable in the SettingsService. do you want it to be a getter? There's no data encapsulation happening

Comment (muff1nman) [reply airsonic-main/src/main/java/org/airsonic/player/service/UPnPService.java]: Yes we should be consistent with all settings. In addition, we should be fetching from the settings service for the value, not from the environment.

**Comment by randomnicode** [comment]
@muff1nman moved port to SettingsService

File: airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java [hunk 2]
```diff
@@ -290,6 +292,10 @@ private static String getFileSystemAppName() {
     public static String getDefaultJDBCUrl() {
         return "jdbc:hsqldb:file:" + getAirsonicHome().getPath() + "/db/" + getFileSystemAppName();
     }
+    
+    public static int getDefaultUPnpPort() {
+        return Optional.ofNullable(System.getProperty("UPNP_PORT")).map(x -> Integer.parseInt(x)).orElse(DEFAULT_UPNP_PORT);
```
Comment (fxthomas) [inline airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java]: For consistency with all other settings, could you use the same kind of casing (e.g. `UpnpPort`) as the other keys and store it at the top of the file in a constant?

**Pull request merged**